int32 current_state
int32 lane_number
float64 dist

